desed_task/dataio/datasets.py

```diff
@@ -147,14 +152,13 @@ def __len__(self):
     def __getitem__(self, item):
         file = self.examples_list[item]
         c_ex = self.examples[file]
-        mixture, fs = sf.read(c_ex["mixture"])
-        mixture = to_mono(mixture, self.random_channel)
+        mixture, fs = torchaudio.load(c_ex["mixture"])
+        if not self.multisrc:
+            mixture = to_mono(mixture, self.random_channel)
```

Comment:
We have duplicated code for each beginning of __getitem__, however doing an abstract class just for that is maybe not useful. 
An idea could be to put this piece of code in a function called `_get_mixture` or `_get_audio` so it is easy to see they are the same, what do you think ?

Reply:
I made a function